packages/vscode-ui5-language-assistant/.vscodeignore

```diff
@@ -12,11 +12,19 @@ node_modules
 !**/@ui5-language-assistant/language-server/src/**
 
 # The VSCode Extension's dev artifacts
+src/**
```

Comment:
these may be useful for debugging flows (seeing the .ts source code in debugger instead of the .js)

Reply:
although I am not sure if this works in the specific case of debugging a vsix